srcpkgs/itch/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'itch'
+pkgname=itch
+version=1.18.0
+revision=1
+archs="i686 x86_64"
```

Comment:
I don't think it needs to be restricted, let people install games wherever they want, they can have fun trying to run them :)

Reply:
Only these two architectures are supported in the project code to compile.